app/javascript/packs/Components/Filters.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import SelectFilter from './SelectFilter'
+import style from './Filters.module.scss'
+
+class Filters extends React.Component {
+    constructor(props) {
+        super(props);
+        this.handleNameChange = this.handleNameChange.bind(this);
+        this.handleRatingChange = this.handleRatingChange.bind(this)
+        this.handleCuisineChange = this.handleCuisineChange.bind(this);
+        this.handleSpeedChange = this.handleSpeedChange.bind(this);
+        this.state = {
+            name: '',
+            rating: 0,
+            cuisine: '',
+            speed:Number.MAX_SAFE_INTEGER
+        };
+    }
```

Comment:
You can skip defining this constructor if you define the state and functions like so:
```jsx
class Filters extends React.Component {
  state = {
    ...
  };

  handleNameChange = o => { ... };

  handleSpeedChange = speed => { ... };

  // etc.
};
```